security-privacy.html

```diff
@@ -207,90 +211,161 @@
-      <p>
-        If Web NFC tags become popular, more and more tags may be added the
-        special records identifying Web NFC tags, making them compatible with
-        the Web NFC API. This is completely fine, once the creator of the tag
-        understands to risks associated with accessing the tags from untrusted
-        web pages.
-      </p>
-    </section>
-    <section> <h3>Web NFC white list for reading</h3>
-      <p>
-        In addition to an identification record, Web NFC tags MAY also contain
-        additional special NDEF records containing a list of URL's as payload.
-        The preferred type of this NDEF record is NFC Forum External Type
-        (TNF=0x04), with the External Type field set to e.g.
-        <code>urn:nfc:ext:w3.org:web-nfc-wl-read</code>.
-        The payload is a string list of URL's, delimited by white space.
-      </p>
-    </section>
-    <section> <h3>Web NFC white list for writing</h3>
-      <p>
-        Web NFC tags MAY also contain another special NDEF record containing a
-        list of URL's permitted to write to the tag.
-        The preferred type of this NDEF record is NFC Forum External Type
-        (TNF=0x04), with the External Type field set to e.g.
-        <code>urn:nfc:ext:w3.org:web-nfc-wl-write</code>.
-        The payload is a string list of URL's, delimited by white space.
-      </p>
-    </section>
-    <section> <h3>Web NFC black list for reading</h3>
-      <p>
-        Web NFC tags MAY also contain another special NDEF record containing a
-        list of URL's specifically NOT permitted to read to the tag.
-        The preferred type of this NDEF record is NFC Forum External Type
-        (TNF=0x04), with the External Type field set to e.g.
-        <code>urn:nfc:ext:w3.org:web-nfc-bl-read</code>.
-        The payload is a string list of URL's, delimited by white space.
-      </p>
-    </section>
-    <section> <h3>Web NFC black list for writing</h3>
-      <p>
-        Web NFC tags MAY also contain another special NDEF record containing a
-        list of URL's specifically NOT permitted to write to the tag.
-        The preferred type of this NDEF record is NFC Forum External Type
-        (TNF=0x04), with the External Type field set to e.g.
-        <code>urn:nfc:ext:w3.org:web-nfc-bl-write</code>.
-        The payload is a string list of URL's, delimited by white space.
-      </p>
-    </section>
+  </section>
+  <section> <h3>Web NFC using encoded metadata</h3>
+    <p>
+      Web NFC may also define an own NDEF extended record format,
+      where the preferred type of this NDEF record is NFC Forum External Type
+      (TNF=0x04), with the External Type field set to
+      <code>urn:nfc:ext:w3.org:webnfc</code>. The payload type is
+      <code>application/webnfc+json</code> (encoded as UTF-8 by default). All
+      data is JSON, with the following schema:
+      <ul>
+        <li>
+          <code>id</code>: <code>"string"</code>; contains the Web NFC
```

Comment:
There is a slight chance there are NFC tags out there with a given content that coincides with that of a Web NFC tag. This minimizes that risk. Otherwise IMO it would be enough to have the web-nfc specific urn namespace in the external type. The answer is: not really needed, but I am waiting on feedback from @sicking on this.